### PR TITLE
Finished problem 23.

### PR DESCRIPTION
**Pull request by ikhat**
Pretty simple, not much to say here I think.

### CODE REVIEW

File: problemsets-mat223/linearalbegra.tex [hunk 1]
```diff
@@ -2013,13 +2013,31 @@ \section*{Dot Product}
...
-		for $A$.
+		\item Draw $\vec u=\mat{2\\3}$ and \emph{all} vectors orthogonal to it. 
+			Call this set $A$.
+			\begin{solution}
+				XXXX
+			\end{solution}
+		\item If $\vec x=\mat{x\\y}$ and $\vec x$ is orthogonal to $\vec u$, 
+			what is $\vec x\cdot \vec u$?
+			\begin{solution}[inline]
+				$\vec x\cdot \vec u = 0$, by definition of orthogonality.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: by *the* definition of orthogonality

File: problemsets-mat223/linearalbegra.tex [hunk 2]
```diff
@@ -2013,13 +2013,31 @@ \section*{Dot Product}
...
+			\end{solution}
+		\item Expand the dot product $\vec u\cdot \vec x$ to get an equation for $A$.
+			\begin{solution}
+				$A$ is the line with vector equation $\vec x = t\mat{3\\2}$. 
+
+				If $\vec x=\mat{x\\y} \in A$, then we have:
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: I prefer without the colon.

File: problemsets-mat223/linearalbegra.tex [hunk 3]
```diff
@@ -2013,13 +2013,31 @@ \section*{Dot Product}
...
+				XXXX
+			\end{solution}
+		\item If $\vec x=\mat{x\\y}$ and $\vec x$ is orthogonal to $\vec u$, 
+			what is $\vec x\cdot \vec u$?
+			\begin{solution}[inline]
+				$\vec x\cdot \vec u = 0$, by definition of orthogonality.
+			\end{solution}
+		\item Expand the dot product $\vec u\cdot \vec x$ to get an equation for $A$.
+			\begin{solution}
+				$A$ is the line with vector equation $\vec x = t\mat{3\\2}$.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Missing a minus sign?

File: problemsets-mat223/linearalbegra.tex [hunk 4]
```diff
@@ -2013,13 +2013,31 @@ \section*{Dot Product}
...
+		\item Expand the dot product $\vec u\cdot \vec x$ to get an equation for $A$.
+			\begin{solution}
+				$A$ is the line with vector equation $\vec x = t\mat{3\\2}$. 
+
+				If $\vec x=\mat{x\\y} \in A$, then we have:
+				\[
+					\vec x\cdot\vec u = 2x + 3y = 0.
+				\]
+				The vector $\mat{3\\2}$ satisfies this equation, as does any
+				other vector if and only if it is a multiple of this one.
```
Comment (siefkenj) [inline problemsets-mat223/linearalbegra.tex]: Unneeded sentence.

File: problemsets-mat223/linearalbegra.tex (hunk 3 above)
Comment (ikhat) [reply problemsets-mat223/linearalbegra.tex]: Oops. Sorry about that. Forgot the minus sign both times.

**Pull request merged**